test/unit/mailer_test.rb

```diff
@@ -23,6 +24,24 @@ class MailerTest < ActiveSupport::TestCase
     end
   end
 
+  context "sending mail for mfa required soon announcement" do
```

Comment:
FYI: There's a few more tests I plan on adding tomorrow. In total, there's 4 cases I want to test for:
* send mail to users with with more than 180M+ downloads and have MFA disabled
* send mail to users with with more than 180M+ downloads and have weak MFA (aka `ui_only`)
* not send mail to users with with more than 180M+ downloads and have strong MFA
* not send mail to users with with less than 180M downloads

Reply:
I was unable to debug why having all 4 tests led to flaky tests. @jchestershopify and I paired on debugging it but haven't come to any concrete conclusions. Given the impending deadline, we'll keep the one integration test in this PR as-is. It tests the "happy path". 

Separately, we'll be exploring this further and will backfill for those other cases once we determine why they're flaky.

Reply:
This is resolved. The [follow-up PR](https://github.com/rubygems/rubygems.org/pull/3167#issuecomment-1207334012) is now closed. The full test suite has been added in.